CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## 0.13.5-dev
 
 * Allow async callbacks in RetryClient.
+* Use callback returned Response.request instead of the passed request to give
```

Comment:
On its own this entry is not easy to understand, maybe just add a mention that this is for the MockHttpClient

Reply:
Done